apple/internal/rule_attrs.bzl

```diff
@@ -712,6 +712,17 @@ def _simulator_runner_template_attr():
         ),
     }
 
+def _locales_to_include_attr():
+    """Returns the attribute required to support configuring the explicit set of locales supported for the bundle."""
+    return {
+        "locales_to_include": attr.string_list(
+            mandatory = False,
+            doc = """
+A list of locales to include in the bundle. Only *.lproj directories that are matched will be copied as a part of the build.
+""",
```

Comment:
Can this mention the `--define` and how it interacts with it?

Reply:
Yeah will add a note.